device/celestica/x86_64-cel_seastone-r0/plugins/psuutil.py

```diff
@@ -18,33 +20,41 @@ def __init__(self):
         PsuBase.__init__(self)
         # DX010 PSU pin mapping
         self.psu = [
-            {'base':216},           # Reserved
+            {'base': self.get_gpio_base()},
             {'abs':27, 'power':22},
             {'abs':28, 'power':25}
         ]
 
+    def get_gpio_base(self):
+        sys_gpio_dir = "/sys/class/gpio"
+        try:
+            for r in os.listdir(sys_gpio_dir):
+                if "gpiochip" in r:
+                    return int(r[8:],10)
+        except Exception as error:
+            logging.error("Unable to get gpio base")
+
+
     def init_psu_gpio(self, pinnum):
```

Comment:
this init_psu_gpio should move to your driver, we should not do init psu gpio on-the-fly.

Reply:
@pphuchar Please add this to our platform driver